src/app/view/components/_common/dot-field-validation-message/dot-field-validation-message.html

```diff
@@ -1 +1 @@
-<div *ngIf="field.dirty && !field.valid" class="ui-messages-error">{{ message }}</div>
+<div *ngIf="field.enabled && field.dirty && !field.valid" class="ui-messages-error">{{ message }}</div>
```

Comment:
I added this, so validation do not show if the form control is disable. would you agree ? @fmontes